cli/main.rs

```diff
@@ -701,6 +701,21 @@ async fn run_command(
     return run_from_stdin(flags).await;
   }
 
+  if flags.to_permission_args().is_empty()
+    && flags.argv.iter().any(|arg| {
+      arg.contains("--allow-env")
```

Comment:
Maybe `arg == "--allow-env"` because that won't require searching the entire `arg` string for each of these.

Reply:
Oh, true. Then `starts_with`